packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -90,6 +90,7 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
         className={this._classNames.root}
         style={viewport ? { minWidth: viewport.width } : {}}
         onClick={this._onHeaderClick}
+        aria-expanded={group ? !group.isCollapsed : undefined}
```

Comment:
```suggestion
        aria-expanded={!group.isCollapsed}
```

We're guaranteed to have a `group` at this point because of the early return at https://github.com/OfficeDev/office-ui-fabric-react/blob/6fcf7c822401e334ba3bbc05a2ed5dd32936708b/packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.base.tsx#L86

Otherwise LGTM!